raaghu-layouts/src/rds-comp-app-shell/rds-comp-app-shell.tsx

```
@@ -2,41 +1,46 @@
   Relaxing = "Relaxing",
   TopNav = "Top Nav",
   SideNav = "Side Nav",
   DoubleNav = "Double Nav",
   OneThreeOne = "1-3-1"
 }
-
+ 
+ 
 const RdsCompAppShell = (props: RdsCompAppShellProps) => {
+ 
   return (
```

ESLint

[error] Expected indentation of 4 spaces but found 2.

(indent)